### ISSUE TITLE
Cloud SQL PostgreSQL sample fails with  [CRITICAL] WORKER TIMEOUT when deployed to Cloud Run

### ISSUE THREAD

**Issue by kurdybacha** [GoogleCloudPlatform/python-docs-samples#4545]
## In which file did you encounter the issue?

https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/cloud-sql/postgres/sqlalchemy

## Describe the issue

Follow the README.
- add `roles/cloudsql.client` to `PROJECT_NUMBER-compute@developer.gserviceaccount.com`
- build sample's image
- deploy cloud run service
- update service with db instance connection name, db user, db pass, db name.

Observer Cloud Run logs for run-sql service.

Expected: service starts
Result:
```
2020-08-24 14:10:00.426 BST[2020-08-24 13:10:00 +0000] [1] [INFO] Starting gunicorn 20.0.4
2020-08-24 14:10:00.428 BST[2020-08-24 13:10:00 +0000] [1] [INFO] Listening at: http://0.0.0.0:8080 (1)
2020-08-24 14:10:00.428 BST[2020-08-24 13:10:00 +0000] [1] [INFO] Using worker: threads
2020-08-24 14:10:00.489 BST[2020-08-24 13:10:00 +0000] [3] [INFO] Booting worker with pid: 3
2020-08-24 14:10:32.178 BST[2020-08-24 13:10:31 +0000] [1] [CRITICAL] WORKER TIMEOUT (pid:3)
2020-08-24 14:10:32.978 BST[2020-08-24 13:10:32 +0000] [3] [INFO] Worker exiting (pid: 3)
```

Locally with Cloud SQL proxy sample works fine (eliminating SQL config issue).

**Comment by engelke** [comment]
I did the steps as shown in the README, and first got an error with the gcloud command that called the service run-mysql. When I fixed it to the name used elsewhere, run-sql, it all worked properly.

The worker timeout might have been an intermittent problem, or perhaps the SQL instance hadn't finished deploying, or had a problem deploying. Feel free to open this issue with more details if needed, but I'm going to fix the typo in README and close this issue.
